contracts/IdleTokenFungible.sol

```diff
@@ -463,9 +452,10 @@ contract IdleTokenFungible is Initializable, ERC20Upgradeable, ReentrancyGuardUp
       _updateFeeInfo();
       uint256 _unclaimedFees = unclaimedFees;
       if (_unclaimedFees > 0) {
-        // send fees
-        _mint(feeAddress, _unclaimedFees * ONE_18 / _tokenPrice());
-        // reset fee counter
+        // send fees (lastNAV was just updated in _updateFeeInfo)
+        _mint(feeAddress, _unclaimedFees * totalSupply() / lastNAV);
+        // reset fee counter and update lastNAV
+        lastNAV += _unclaimedFees;
```

Comment:
`lastNAV` is NAV without that are yet to be accounted. so, in rebalance we add `_unclaimedFees` to be claimed, right ?

Reply:
`lastNAV` have all fees removed in general and this should still be valid here so, after minting new tranche tokens at line 456, we add that amount to the `lastNAV` in the same way as we do in `mintIdleToken`. This is because the minted amount `_unclaimedFees * totalSupply() / lastNAV` is 'backed' by `_unclaimedFees` that are now included in the NAV like every other deposit

Reply:
got it